.github/integration/tests/postgres/60_mapper_queries.sh

```diff
@@ -6,14 +6,39 @@ export PGPASSWORD=mapper
 ## map file to dataset
 accession="urn:uuid:7964e232-8830-4351-8adb-e4ebb71fafed"
 dataset="urn:neic:ci-test-dataset"
-file_id=$(psql -U mapper -h postgres -d sda -At -c "SELECT file_id from local_ega.archive_files WHERE stable_id = '$accession';")
-if [ "$file_id" -ne 1 ]; then
+
+file_id=$(psql -U mapper -h postgres -d sda -At -c "SELECT id FROM sda.files WHERE stable_id = '$accession';")
+if [ "$file_id" = "" ]; then
```

Comment:
Nitpick from me:

```suggestion
if [ -z "$file_id" ]; then
```

And there _is_ a nicer and more secure way of inserting the values from shell variables into SQL statements with `psql` that does not involve creating SQL injections for each and every one of them. However, we might not be interested in that here?

Reply:
Once everything is merged and all tests are created we might go through them and do some cleanup.